python/tskit/drawing.py

```diff
@@ -467,19 +506,23 @@ def draw(self):
                     labels = self.right_labels
                 else:
                     labels = self.left_labels
-            # TODO add ID to node label text.
             # TODO get rid of these manual positioning tweaks and add them
```

Comment:
Oh, sorry, I added the thumbsup, but misunderstood. The offsets aren't in a transform at the moment: we still have right_labels, mid_labels, & left_labels. It would be better to add classes, I think, but I don't want to fix this right now, so am leaving it as-is. Hence the manual-positioning TODO still stands

Reply:
OK I get it now!